daemon/protocol/python/scoot/client_lib.py

```
@@ -246,3 +292,17 @@ def domain(status):
                  exit_code=status.exit_code,
                  error=status.error)
   return [domain(x) for x in resp.status]
+
+
+def stop_daemon():
+  """ Request that the daemon be Stopped
+  """
+  global _client
+  if not is_started():
+    raise ScootException(Exception("Connection not started."))
+  req = daemon_pb2.EmptyStruct()
+  try:
+    _client.StopDaemon(req)
+  except Exception:
```

do we really want to swallow all errors without any logging/handling?